functions/htmlcache.php

```diff
@@ -20,6 +20,10 @@ function htmlcache_filename($withDirectory = true)
             $protocol = 'https://';
         }
 
+        if (!isset($_SERVER['HTTP_HOST'])) {
+            return false;
```

Comment:
I added this because [schematic](https://github.com/nerds-and-company/schematic) was borking since `HTTP_HOST` doesn't get defined.